recipes/gunicorn/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "gunicorn" %}
+{% set version = "19.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://github.com/benoitc/gunicorn/archive/{{ version }}.tar.gz
+  sha256: 1e0de4957bea60bfcff5215664bdfc3cf02c78e2aae9586766a9b4b437aebbb0
+  patches:
+    # See https://github.com/conda-forge/staged-recipes/pull/885#discussion_r68602712
+    # git diff --no-prefix -- gunicorn/workers/_gaiohttp.py /dev/null
+    - no_gaiohttp.patch  # [py2k]
```

Comment:
I can't seem to link to the discussion (GitHub is bad about this 😒). What is up with this patch?

Also, did this get referred upstream? If not, we should do this. If so, a reference to that issue/PR would be nice.